extensions/blocks/map/deprecated/v1/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { omit } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import save from './save';
+
+const attributes = {
+	align: {
+		type: 'string',
+	},
+	points: {
+		type: 'array',
+		default: [],
+	},
+	mapDetails: {
+		type: 'boolean',
+		default: true,
+	},
+	zoom: {
+		type: 'integer',
+		default: 13,
+	},
+	mapCenter: {
+		type: 'object',
+		default: {
+			longitude: -122.41941550000001,
+			latitude: 37.7749295,
+		},
+	},
+	mapStyle: {
+		type: 'string',
+		default: 'default',
+	},
+	markerColor: {
+		type: 'string',
+		default: 'red',
+	},
+	preview: {
+		type: 'boolean',
+		default: false,
+	},
+	scrollToZoom: {
+		type: 'boolean',
+		default: false,
+	},
+	mapHeight: {
+		type: 'integer',
+	},
+	showFullscreenButton: {
+		type: 'boolean',
+		default: true,
+	},
+};
+
+export default {
+	attributes,
+	migrate: oldAttributes => omit( oldAttributes, 'mapStyle' ),
```

Comment:
This way we are losing the `mapStyle` choice when migrating a block, reverting it to the default style.

To repro, try adding a Map block with style "satellite" on master, then checkout this PR, reload the post and observe how the map is now with style "default".

One way to fix this might be to convert `mapStyle` to the `is-style-` class:

```es6
migrate: oldAttributes => ( {
	...omit( oldAttributes, 'mapStyle' ),
	className: `${ oldAttributes.className } is-style-${ oldAttributes.mapStyle }`,
} ),
```

Reply:
👍🏻 1253f52ae